src/ext/bitbuffer.ts

```diff
@@ -148,6 +152,28 @@ BitStream.prototype.readBitCoord = function(this: BitStream) {
   return value;
 };
 
+BitStream.prototype.readUVarInt32 = function(this: BitStream) {
```

Comment:
It comes from https://github.com/ValveSoftware/csgo-demoinfo/blob/master/demoinfogo/demofilebitbuf.cpp#L343